popup.css

```
@@ -31,3 +31,14 @@ button:hover {
     color: #fff;
     cursor: pointer;
 }
+
+#version-div {
+    margin: 10px 0 10px 0;
```

```suggestion
    text-align: center;
    margin: 10px 0 10px 0;
```
Maybe we could add this so it's a bit nicer? :)

---

Yup, happy to do that